ckan_cloud_operator/providers/storage/azure/constants.py

```diff
@@ -0,0 +1 @@
+PROVIDER_ID = 'azure'
```

Comment:
use a more accurate provider id ('azure' is already in use)

Reply:
I'll rename it to `abs` (for Azure Blob Storage)

Reply:
please resolve only when done...

Reply:
done